lando/worker/cwlworkflow.py

```diff
@@ -3,9 +3,90 @@
 """
 from __future__ import print_function
 import os
+import shutil
 import urllib
+import datetime
+import json
 from subprocess import PIPE, Popen
 from lando.exceptions import JobStepFailed
+from lando.worker.cwlreport import create_workflow_info, CwlReport
+
+
+RUN_CWL_COMMAND = "cwl-runner"
+RUN_CWL_OUTDIR_ARG = "--outdir"
+
+REPORT_FILENAME = 'Bespin-Report.txt'
+
+LOGS_DIRECTORY = 'logs'
+JOB_STDOUT_FILENAME = 'cwltool-output.json'
+JOB_STDERR_FILENAME = 'cwltool-output.log'
+JOB_DATA_FILENAME = 'job-data.json'
+
+WORKFLOW_DIRECTORY = 'workflow'
+WORKFLOW_FILENAME = 'workflow.cwl'
+JOB_ORDER_FILENAME = 'workflow.yml'
```

Comment:
Files named `workflow.cwl` and `workflow.yml` aren't very descriptive. Even as someone with CWL experience I find this confusing - At a glance I can't tell which is the workflow.

Why not carry forward a descriptive name for the workflow (currently `rnaseq-pt1.cwl`) and something like `job-34-input.yml` for the job input file?

Also, is the job order YAML or JSON? I thought we stored it as JSON in bespin-api